### DISCUSSION TITLE
Best way to cable a microphone 15m away from RPi?

### DISCUSSION THREAD

**Discussion by srchild** [mcguirepr89/BirdNET-Pi#277] [Q&A]
I want to move my microphone to a better position, away from human noise. 

I've identified my intended location which has some shelter to protect the microphone from wet weather. (We live on moorlands and get a lot of horizontal driving rain). It will involve running a cable approximately 15m outside to reach a canopy over a back door. (cables will run around the outside walls of the building, not crossing any pathways or roads or anything like that). What is the best configuration?

Currently I'm using a Rode Smartlav+ (TRRS) with the purple TRRS Ugreen adapter. I also have available (amongst others) a Rode Lavalier with Micon-5 (balanced XLR connector which converts phantom 48V to a lower voltage for the microphone). I also just ordered the Boya M3 USB microphone as I saw it cheap on eBay.

1. I've tested simply inserting a few metres of shielded audio cable between microphone and the Ugreen adapter, and that introduced a lot of noise whether I connected the shield at both ends or only at the Ugreen end.

2. So I'm thinking I should use a balanced XLR cable with Rode lavalier / Micon5 for an extension as long as 15m. But that raises the question of supplying phantom 48V which the Ugreen cannot offer. I'll need to buy some sort of USB audio interface which can support phantom power and balanced XLR (e.g. Focusrite Scarlett Solo or 2i2, or some other brand such as Behringer which has mixed reviews). Or I could use a simpler USB/XLR adapter cable and use a passthrough XLR box which adds phantom power (but which may also add noise?). So not a very simple solution.

3. Perhaps I should buy a POE hat for my RPi, and run an ethernet cable to my microphone location and site both microphone and RPi undercover outside? One disadvantage of this, besides the risk of RPi getting wet, is that it will not be possible to position the microphone more than 70cm from the RPi, and so the microphone may pick up radio emissions from RPi?

4. Perhaps, instead, I should buy a 15m long USB extension cable, and place both the microphone and Ugreen adaptor at the remote location? Will a USB cable that long perform well enough?

5. Is there a #5? Your suggestions please!

Thanks

Simon

**Comment by srchild** [comment]
I'm leaning towards (3) the POE solution. It seems the most likely to work. It avoids the expense of a USB audio interface and the potential unreliability of a long USB cable. I have lots of Cat5e cable and can easily run a length of it to my location. I have a POE switch already.

I've found an external POE splitter (not a hat), which is half the price of a hat and will work with my current case

[https://www.amazon.co.uk/DSLRKIT-Gigabit-Active-Splitter-Raspberry/dp/B07W8M2GX4](https://www.amazon.co.uk/DSLRKIT-Gigabit-Active-Splitter-Raspberry/dp/B07W8M2GX4)

**Comment by mcguirepr89** [comment]
> 5. Is there a #5? Your suggestions please!

Yes, [that is enabled to 2 threads by default](https://github.com/mcguirepr89/BirdNET-Pi/blob/73707817a2e314dfb34269e5117d92e553236811/scripts/server.py#L65). I don't have good answers for the other bits, but I'm sure someone will help guide you in the right direction.

**Reply by srchild** [reply]
Ah, I wrote (using a pound sign, which apparently was not a good idea) "Is there a number 5?" i.e. any options that I'm overlooking. But the pound sign triggered a link to discussion thread number 5!

**Reply by mcguirepr89** [reply]
Hahaha I think I understand what happened, now!! hahaha

**Comment by srchild** [comment]
Update:

I went the POE route and took the risk of housing my Pi under an open back door porch. I ran a Cat5e cable around the outside wall to reach it. I used [one of these active POE splitters from Amazon](https://www.amazon.co.uk/DSLRKIT-Gigabit-Active-Splitter-Raspberry/dp/B07W8M2GX4). That is half the price of a POE hat and saves the trouble of changing to a hat compatible case, and I had no trouble with it - it worked well.

However I felt a bit anxious housing the Pi outside even in the porch as we do get driving rain with strong winds in this area. The ethernet cable was wet only about 7-10cm below the Pi. Also, the microphone was picking up a little fan noise from the nearby Pi.

I found a way to run the ethernet cable through one side of the back door frame, and the USB extension cable through the other side of the door frame, so that I could house the Pi indoors while still connecting to the outside microphone. It was easier for me than it might have been to run cables through the back door frame as that door and frame have recently been replaced and they are not yet properly sealed! I just had to poke a couple of holes in the builder's foam :)

Previously the microphone was poking out of my server room window, adjacent to my home office. Now that the microphone is away from machine/phone/keyboard/human noises it is picking up much less ambient sound. Also I have changed the foam windshield to a deadcat-style fluffy windshield which is supposed to reduce wind noise more effectively. Hopefully the bird identifications will be more numerous / more accurate now...

As examples, I attach before and after spectrograms from the previous and present location. Notice how the low frequency background noise is now much less.

Now, what can I do to stop my microphone from getting wet which won't reduce the sound reaching it??
![before](https://user-images.githubusercontent.com/104400455/170841470-33df3e9a-e420-4569-a447-3be2f1a0c051.png)
![after](https://user-images.githubusercontent.com/104400455/170841482-74aa972a-c88b-461d-9592-b2ad9e1b03c8.png)

**Comment by alexbelgium** [comment]
> I want to move my microphone to a better position, away from human noise.
> 
> I've identified my intended location which has some shelter to protect the microphone from wet weather. (We live on moorlands and get a lot of horizontal driving rain). It will involve running a cable approximately 15m outside to reach a canopy over a back door. (cables will run around the outside walls of the building, not crossing any pathways or roads or anything like that). What is the best configuration?
> 
> Currently I'm using a Rode Smartlav+ (TRRS) with the purple TRRS Ugreen adapter. I also have available (amongst others) a Rode Lavalier with Micon-5 (balanced XLR connector which converts phantom 48V to a lower voltage for the microphone). I also just ordered the Boya M3 USB microphone as I saw it cheap on eBay.
> 
> 1. I've tested simply inserting a few metres of shielded audio cable between microphone and the Ugreen adapter, and that introduced a lot of noise whether I connected the shield at both ends or only at the Ugreen end.
> 2. So I'm thinking I should use a balanced XLR cable with Rode lavalier / Micon5 for an extension as long as 15m. But that raises the question of supplying phantom 48V which the Ugreen cannot offer. I'll need to buy some sort of USB audio interface which can support phantom power and balanced XLR (e.g. Focusrite Scarlett Solo or 2i2, or some other brand such as Behringer which has mixed reviews). Or I could use a simpler USB/XLR adapter cable and use a passthrough XLR box which adds phantom power (but which may also add noise?). So not a very simple solution.
> 3. Perhaps I should buy a POE hat for my RPi, and run an ethernet cable to my microphone location and site both microphone and RPi undercover outside? One disadvantage of this, besides the risk of RPi getting wet, is that it will not be possible to position the microphone more than 70cm from the RPi, and so the microphone may pick up radio emissions from RPi?
> 4. Perhaps, instead, I should buy a 15m long USB extension cable, and place both the microphone and Ugreen adaptor at the remote location? Will a USB cable that long perform well enough?
> 5. Is there a [multithreading #5](https://github.com/mcguirepr89/BirdNET-Pi/issues/5)? Your suggestions please!
> 
> Thanks
> 
> Simon

Hi ! In the end which mic did you end up using and was the best? Thanks!

**Reply by srchild** [reply]
I'm still configured as my message above: https://github.com/mcguirepr89/BirdNET-Pi/discussions/277#discussioncomment-2841817 and I am happy with it. My microphone is keeping dry and my recording is working.

Smartlav+ outside, keeping dry under an open porch canopy, and connected to the Ugreen USB interface. USB cable enters at the side of the outside doorframe and connects to the Pi4 inside.

**Reply by alexbelgium** [reply]
Thanks very much! I've ordered exactly what you had and am now waiting for delivery :-)

**Reply by srchild** [reply]
The Boya offered the convenience of being a USB microphone and so not needing a USB interface. I can't remember now but I think I tried it only briefly and it seemed noisier than the Smartlav+. That wasn't surprising as the Smartlav+ is a higher-spec and more expensive microphone (but can be bought used for a reasonable price on eBay).

I have two BirdNET-Pi installations here, both same configuration, Pi4 2Gb, Smartlav+ with ugreen interface, microphone outside but sheilded from weather. Of course there is some overlap between the results of the two, but there is also some difference. One on the West side of the building faces open moorland. The second is facing East by a stream with ponds and with bushes along the bank, so picks up more warblers and water birds (though in wet weather the sound of flowing water is loud).
